lessons/01-setting-up.md

```diff
@@ -11,6 +11,7 @@ up our project.
 ```
 git clone <tutorial url>
 cd react-router-tutorial
+npm install
 git checkout start
```

Comment:
`npm install` should probably come after `git checkout` so it is performed on the correct branch?